src/glossary.rst

```diff
@@ -967,3 +967,115 @@ Glossary
       See also:
 
       * :ref:`Graph Branching`
+
+source workflow
+   A source workflow is a :term:`cylc workflow` that has not yet been
+   :term:`installed` into the :term:`run directory`.
+
+workflow
+cylc workflow
+   A Cylc workflow is a collection of things to run (:term:`tasks`) along with
+   the constraints that govern the order in which they run
+   (:term:`dependencies`) represented in Cylc format in a
+   :cylc:conf:`flow.cylc` file.
+
+   For example here is a Cylc workflow representing the brewing process:
+
+   .. code-block:: cylc
+      :caption: flow.cylc
+
+      [scheduling]
+          cycling mode = integer
+          initial cycle point = 1
+          [[graph]]
+              # repeat this for each batch
+              P1 = """
+                  # the stages of brewing in the order they must occur in
+                  malt => mash => sparge => boil => chill => ferment => rack
+
+                  # must finish the sparge of one batch before
+                  # starting the next one
+                  # sparge[-P1] => mash
+              """
+
+.. TODO - add this after the universal ID work
+
+   workflow id
+      A workflow ID is the name Cylc uses to identify a :term:`workflow` in
+      the :term:`run directory`.
+
+      This ID is used on the command line and in the GUI.
+
+      The ID is the relative path between the :term:`workflows <workflow>`
+      :term:`run directory` and the :term:`cylc run directory`.
+
+      For example, if your workflow is in ``~/cylc-run/foo/bar/run1`` then its ID
+      is ``foo/bar/run1``.
+
+      .. ..note:: For convenience you can omit the ``/run1`` on the command line.
+
+flow
+   A flow is a single logical run of a :term:`workflow` that is done by
+   a :term:`scheduler`.
+
+   A flow can be :term:`played <play>` and :term:`paused <pause>`,
```

Comment:
I don't think there is a defintion for "play" at the moment, I suppose it's "causing the scheduler to run, either by starting it, restarting it or resuming it"

Reply:
I think there are a few definitions missing ATM, waiting to get the "flow" stuff sorted before filling in the blanks.

Reply:
Would these emoji's potentially make our conceit RE: The control terminology more obvious? ⏯️ ⏹️